src/implementations/twig/components/site-header-core/site-header-core.story.js

```diff
@@ -35,17 +35,15 @@ const getArgTypes = () => {
     },
   };
 
-  if (system === 'eu') {
-    argTypes.menu = {
-      type: { name: 'boolean' },
-      defaultValue: true,
-      description: 'Toggle menu visibility',
-      table: {
-        type: { summary: 'object' },
-        defaultValue: { summary: '{}' },
-      },
-    };
-  }
+  argTypes.menu = {
+    type: { name: 'boolean' },
+    defaultValue: true,
+    description: 'Toggle menu visibility',
+    table: {
```

Comment:
Small thing, but i think that when the controls are nothing related to the data we should simply remove any information from the "table", it doesn't make sense to specify the data type of the control and the default value..

Reply:
Thks @planctus, PR updated.